docs/api/api.md

```diff
@@ -46,10 +48,12 @@ JSON parameters:
   - transaction - user-defined transaction that executed by the Saga.
     - method - user-defined, HTTP method.
     - path - user-defined, HTTP path.
+    - retries - int, optional, default -1. The max retry times for transaction. If this parameter is less than or equal to 0, transaction will retry infinitely.
     - params - support `form`,`json`,`body`,`query`.
   - compensation - user-defined compensation that executed by the Saga.
     - method - user-defined, HTTP method.
     - path - user-defined, HTTP path.
+    - retries - int, optional, default 3. The max retry times for compensation.

```

Comment:
Does it support -1 setting?

Reply:
When compensation set retries less than or equal to 0, it will be set default value.
This is the logic that was there, I have not changed it.